CHANGELOG.rst

```diff
@@ -13,9 +13,21 @@ Change Log
 
 Unreleased
 ~~~~~~~~~~
-
 *
 
+[1.0.0] - 2018-08-27
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_______
+
+* Add keys method to RequestCache.
+
+Changed
+_______
+
+* Change is_hit/is_miss to is_found.
```

Comment:
@ormsbee: FYI: Because it wasn't a big deal to update, we decided to do this before switching edx-platform over to use the new interface.